RubberduckTests/Grammar/VBAParserTests.cs

```diff
@@ -1484,6 +1484,126 @@ Sub Test()
             AssertTree(parseResult.Item1, parseResult.Item2, "//forNextStmt");
         }
 
+        [Category("Parser")]
+        [Test]
+        public void TestCombinedForNextStatement()
+        {
+            string code = @"
+Sub Test()
+    For n = 1 To 10
+        For m = 1 To 20
+            a = m + n
+        Next m _
+    , n%
+End Sub";
+            var parseResult = Parse(code);
+            AssertTree(parseResult.Item1, parseResult.Item2, "//forNextStmt", matches => matches.Count == 2);
```

Comment:
I realize the PR is already merged but I want to comment that all tests seems to verify that there are correct counts of the nested for blocks. Shouldn't we have a test to verify that the `m` is the inner and `n` is the outer, too?